apps/src/JsDebuggerUi.js

```diff
@@ -100,12 +93,11 @@ JsDebuggerUi.prototype.attachTo = function (jsInterpreter) {
   this.observer_.observe(jsInterpreter.onExecutionWarning,
       this.log.bind(this));
 
-  this.watchIntervalId_ = setInterval(this.updateWatchView_.bind(this),
-      WATCH_TIMER_PERIOD);
-
   this.updatePauseUiState();
   this.clearDebugOutput();
   this.clearDebugInput();
+  this.watchIntervalId_ = setInterval(this.updateWatchExpressions_.bind(this),
```

Comment:
Ultimately ended up adding this "update" functionality back in here rather than the Watchers UI React file—this made some sense as this module is already where the JS interpreter is registered as a single point of truth for both Game Lab & App Lab.

Now it gets the current set of expressions to evaluate from the Redux store, though.